build/Linux/test/scripts/custom_attributes_test.sh

```diff
@@ -9,7 +9,7 @@ install_agent
 verify_no_logs
 
 CORECLR_ENABLE_PROFILING=0 dotnet build
-dotnet bin/Debug/netcoreapp2.1/custom_attributes.dll
+/usr/local/newrelic-netcore20-agent/run.sh dotnet bin/Debug/net6.0/custom_attributes.dll
```

Comment:
Why the swap to using our script vs the direct call to dotnet?

Reply:
Good catch, that was an attempt at troubleshooting what turned out to be an arm64/amd64 confusion issue.